frontend/src/patientApp/timeOfTest/constants.ts

```diff
@@ -97,8 +97,8 @@ export const syphilisSymptomsMap = {
   "56940005": "Palmar (hand)/plantar (foot) rash",
   "91554004": "Flat white warts",
   "15188001": "Hearing loss",
-  "46636008": "Blurred vision",
```

Comment:
⚠️ Noticed the SNOMED we had originally for "Blurred vision" was incorrect (before 46636008, after [246636008](https://browser.ihtsdotools.org/?perspective=full&conceptId1=246636008&edition=MAIN/2023-09-01&release=&languages=en&latestRedirect=false)) see #8100 description as well